src/templates/actors/character-sheet2e.hbs

```diff
@@ -95,23 +98,89 @@
+          </div>
+        </div>
+        <div class="row data">
+          <div class="column" style="width:25%">
+            <input name="system.house" class="text-entry" type="text" value="{{system.house}}" placeholder="{{localize 'sta.actor.character.klingon.house'}}" />
+          </div>
+          <div class="column" style="width:25%">
+            <input name="system.caste" class="text-entry" type="text" value="{{system.caste}}" placeholder="{{localize 'sta.actor.character.klingon.caste'}}" />
+          </div>
+          <div class="column" style="width:25%">
+              <input name="system.status" class="text-entry" type="text" value="{{system.status}}" placeholder="{{localize 'sta.actor.character.klingon.status'}}" />
+          </div>
+          <div class="column" style="width:25%">
+              <input name="system.legacy" class="text-entry" type="text" value="{{system.legacy}}" placeholder="{{localize 'sta.actor.character.klingon.legacy'}}" />
+          </div>
+        </div>
+        <div class="row title">
+          <div class="column title" style="width:25%">
+            <div> {{localize 'sta.actor.character.klingon.temperament'}} </div>
+          </div>
+          <div class="column title" style="width:45%">
+              <div> {{localize 'sta.actor.character.klingon.househistory'}} </div>
+          </div>
+          <div class="column title" style="width:10%">
+              <div> {{localize 'sta.actor.character.klingon.influence'}} </div>
+          </div>
+          <div class="column title" style="width:10%">
+              <div> {{localize 'sta.actor.character.klingon.might'}} </div>
+          </div>
+          <div class="column title" style="width:10%">
+              <div> {{localize 'sta.actor.character.klingon.wealth'}} </div>
+          </div>
+        </div>
+        <div class="row data">
+          <div class="column" style="width:25%">
+            <input name="system.temperament" class="text-entry" type="text" value="{{system.temperament}}" placeholder="{{localize 'sta.actor.character.klingon.temperament'}}" />
+          </div>
+          <div class="column" style="width:45%">
+            <input name="system.househistory" class="text-entry" type="text" value="{{system.househistory}}" placeholder="{{localize 'sta.actor.character.klingon.househistory'}}" />
+          </div>
+          <div class="column" style="width:10%">
+              <input class="text-entry" name="system.influence" type="number" value="{{system.influence}}" placeholder="0" />
+          </div>
+          <div class="column" style="width:10%">
+              <input class="text-entry" name="system.might" type="number" value="{{system.might}}" placeholder="0" />
+          </div>
+          <div class="column" style="width:10%">
+              <input class="text-entry" name="system.wealth" type="number" value="{{system.wealth}}" placeholder="0" />
+          </div>
+        </div>
+        </div>
+        <div class="row acclaim">
```

Comment:
A roll button seems appropriate for the 2E sheet's Acclaim and Reprimand sections, too.

Reply:
It has replaced the challenge roll button on the 2e sheet, that is a much better space for it.